bonus.js

```diff
@@ -0,0 +1,67 @@
+// Maak drie knoppen op de pagina: Sorteer op prijs, AmbiLight TV's
+// en Uitverkochte exemplaren. Gebruik de code die je in opdracht 1b, 1c en 1d
+// hebt gemaakt en schrijf dit om naar functies zodat je ze kunt aanroepen op het
+// moment dat de buttons geklikt worden. Zorg ervoor dat de functies de uitkomsten
+// in de de console loggen als de gebruiker op de bijbehorende knop klikt.
+// Tip: lees hiervoor paragraaf 7.4 op EdHub eens door!
+
+// Zorg er nu voor, in plaats van dat de uitkomsten in de console worden gelogd,
+// dat de uitkomsten worden meegegeven aan jouw functie uit 4e zodat
+// de resultaten daadwerkelijk op de pagina weergegeven worden!
+function soldTvs(tvs) {
+    return tvs.filter((tv) => {
+        return (tv.originalStock === tv.sold);
+    });
+}
+
+function tvsWithAmbilight(tvs) {
+   return  tvs.filter((tv) => {
+        return tv.options.ambiLight;
+    });
+
+}
+
+function priceSort(tvs) {
+    tvs.sort((a, b) => {
+        if (a.price > b.price) {
+            return 1;
+        }
+        if (a.price < b.price) {
+            return -1
+        }
+        return 0;
```

Comment:
Je kunt dit versimpelen
```suggestion
        return a.price - b.price;
```